pkgs/development/tools/boost-build/default.nix

```diff
@@ -1,11 +1,14 @@
-{ stdenv, fetchurl }:
+{ stdenv, fetchFromGitHub }:
 
 stdenv.mkDerivation rec {
-  name = "boost-build-2.0-m12";
-
-  src = fetchurl {
-    url = "mirror://sourceforge/boost/${name}.tar.bz2";
-    sha256 = "10sbbkx2752r4i1yshyp47nw29lyi1p34sy6hj7ivvnddiliayca";
+  name = "boost-build-2.0-${version}";
```

Comment:
Yes, the "2.0" can be probably dropped, even though it's not really a version number, it's the name of the project after a refactoring effort.

Boost Build releases are not tied to Boost releases, I'm not sure why they have Boost version tags in the repo, but it's probably not a good idea to use them when  Boost Build isn't being bundled with Boost itself.